README.md

```diff
@@ -75,6 +75,8 @@ The package exports a global `Avatar` object which has a property named `options
   - `cssClassPrefix`: This property is used to prefix the CSS classes of the DOM elements generated by this package.
   - `imageSizes`: This property defines the avatar sizes that are available to your project
   - `backgroundColor`: The background color for initials avatars (color name or hex value string). The default is gray (`"#AAA"`).
+  - `useBackgroundColorScheme`: Seting this option to `true` enables background colors for initials as a function of the user's name. This options overrides the `backgroundColor` and by default is set to `false`.
```

Comment:
This seems superfluous to me. You just need to check if backgroundColorScheme is defined instead.